integration_test/gce/gce_testing.go

```diff
@@ -21,55 +21,71 @@ To run a test based on this library, you can either:
 
 * use Kokoro by triggering automated presubmits on your change, or
 * use "go test" directly, after performing the setup steps described
-  in README.md.
+
+in README.md.
 
 NOTE: When testing Windows VMs without using Kokoro, PROJECT needs to be
-    a project whose firewall allows WinRM connections.
-    [Kokoro can use stackdriver-test-143416, which does not allow WinRM
-    connections, because our Kokoro workers are also running in that project.]
+
+	a project whose firewall allows WinRM connections.
+	[Kokoro can use stackdriver-test-143416, which does not allow WinRM
+	connections, because our Kokoro workers are also running in that project.]
```

Comment:
```suggestion
a project whose firewall allows WinRM connections.
[Kokoro can use stackdriver-test-143416, which does not allow WinRM
connections, because our Kokoro workers are also running in that project.]
```